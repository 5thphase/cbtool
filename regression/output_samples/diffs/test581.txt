|Redis Version |3.0.6 
|VMs APPLICATION ERRORS |6 
