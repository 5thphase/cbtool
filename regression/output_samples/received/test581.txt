AIDRS object 5E946709-5E93-5ED1-98AE-AFF3FA383C4F (named "aidrs_1") sucessfully attached to this experiment.
The following ATTACHED AIDRSs are in this experiment (Cloud TESTCLOUD) :
|name                |pattern                |type              
|aidrs_1             |simplenw               |nullworkload      

The AIDRS object aidrs_1, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AIDRS object key)               |value                                
|iait                                       |10                                   
|lifetime                                   |40                                   
|max_ais                                    |8000                                 

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |7.94M                         
|Redis Version                                        |2.8.4                         
|Uptime (in seconds)                                  |4162496                       
|Total Connections Received                           |152833                        
|Total Commands Processed                             |16858070                      
|Number of Keys                                       |670                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |1                             
|AIs                                                  |1                             
|AIDRSs                                               |1                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |1                             
|VMs REPORTED                                         |1                             
|VMs ARRIVED                                          |118                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |117                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|AI RESERVATIONS                                      |1                             
|AIs ARRIVED                                          |29                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |28                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |1                             
|AIDRSs ARRIVED                                       |1                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |387                           

 ---------------- VMC-WIDE COUNTERS ----------------
|705F5EB2-0B93-5D6B-BD27-826010AF7814 (simzone_a) VM RESERVATIONS|1                             
|620676DB-C9FC-5D5A-978C-8AEBECE97E27 (simzone_b) VM RESERVATIONS|0                             
|8DF2EF36-7448-5148-AD1A-F835627AC672 (simzone_d) VM RESERVATIONS|0                             
|4F9D017D-1712-5FB2-8CEB-61527E065DC8 (simzone_c) VM RESERVATIONS|0                             

 ---------------- AIDRS-WIDE COUNTERS ----------------
|5E946709-5E93-5ED1-98AE-AFF3FA383C4F (aidrs_1) AI RESERVATIONS|1                             

