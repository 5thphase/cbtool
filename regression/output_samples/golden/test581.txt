AIDRS object 77085658-C7FF-5DDD-B4FA-F7E629B356C2 (named "aidrs_1") sucessfully attached to this experiment.
The following ATTACHED AIDRSs are in this experiment (Cloud TESTCLOUD) :
|name                |pattern                |type              
|aidrs_1             |simplenw               |nullworkload      

The AIDRS object aidrs_1, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AIDRS object key)               |value                                
|iait                                       |10                                   
|lifetime                                   |40                                   
|max_ais                                    |8000                                 

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |26.14M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |14436399                      
|Total Connections Received                           |614873                        
|Total Commands Processed                             |58970806                      
|Number of Keys                                       |1221                          
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |28                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |1                             
|AIs                                                  |1                             
|AIDRSs                                               |1                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |1                             
|VMs ISSUED                                           |122                           
|VMs ARRIVED                                          |118                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |117                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |7                             
|AI RESERVATIONS                                      |1                             
|AIs ISSUED                                           |31                            
|AIs ARRIVED                                          |29                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |28                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |1                             
|AIDRSs ISSUED                                        |1                             
|AIDRSs ARRIVED                                       |1                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |507                           

 ---------------- VMC-WIDE COUNTERS ----------------
|2C0CBD0C-C4AB-589F-B1DC-473FE1792EEA (simzone_b) VM RESERVATIONS|1                             
|09BE1FD2-5286-5B6E-AA11-D68D33341047 (simzone_a) VM RESERVATIONS|0                             
|95B3CBBD-180B-59D1-9048-B3EAC63E092B (simzone_d) VM RESERVATIONS|0                             
|4DFFE7AE-4E70-5DA7-82EF-D0BC8960A44B (simzone_c) VM RESERVATIONS|0                             

 ---------------- AIDRS-WIDE COUNTERS ----------------
|77085658-C7FF-5DDD-B4FA-F7E629B356C2 (aidrs_1) AI RESERVATIONS|1                             

